clang/include/clang/APINotes/Types.h

```diff
@@ -542,6 +542,9 @@ class FunctionInfo : public CommonEntityInfo {
   /// The result type of this function, as a C type.
   std::string ResultType;
 
+  /// Swift name of this entity.
+  std::string SwiftReturnOwnership;
```

Comment:
Do we need a string to represent this? Wouldn't an enum make more sense?

Reply:
I think it's fine to represent this as a string. This way it's easier to add new cases in Swift without breaking C++ projects that build with older versions of Clang.